app/src/main/java/com/xplore/paymobile/ui/info/InfoFragment.kt

```diff
@@ -26,8 +32,50 @@ class InfoFragment : Fragment() {
         return binding.root
     }
 
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+
+        clientSupportPhoneNumber = getString(R.string.client_support_phone_number)
+        setupTextViews()
+        setupClickListeners()
+    }
+
+    private fun setupTextViews() {
+        binding.apply {
+            callMessage.text = getString(R.string.info_screen_call_message, clientSupportPhoneNumber)
+            appVersion.text = getString(R.string.info_screen_app_version, BuildConfig.VERSION_NAME)
+        }
+    }
+
+    private fun setupClickListeners() {
+        binding.apply {
+            callButton.setOnClickListener {
+                callClientSupport()
+            }
+            termsAndConditionsButton.setOnClickListener {
+                openTermsAndConditionsLink()
+            }
+            logOutButton.setOnClickListener {
+                // TODO to be implemented
+            }
+        }
+    }
+
+    private fun callClientSupport() {
+        val phoneIntent = Intent(Intent.ACTION_CALL)
```

Comment:
i think ACTION_DIAL would be the better option to use. You also don't need any permissions for it, so the permission declaration can be removed